src/Backend/Form/Form.php

```diff
@@ -112,22 +85,27 @@ protected function afterValidation(array $data): array
         return $data;
     }
 
-    public function afterCreate($resource)
+    protected function createResource($data)
     {
-        //
+        $this->resource = $this->resource->newQuery()->create($data);
+        $this->afterCreateResource($this->resource);
     }
 
-    protected function removeNullValues()
+    public function afterCreateResource($resource)
```

Comment:
```suggestion
    protected function afterCreateResource($resource)
```

this method will be overridden in the form class and is not part of its api :-)